lib/multitenancy.ex

```diff
@@ -15,8 +15,6 @@ defmodule AshPostgres.MultiTenancy do
       migrations_path ||
         repo.config()[:tenant_migrations_path] || default_tenant_migration_path(repo)
 
-    Code.compiler_options(ignore_module_conflict: true)
```

Comment:
This configuration is process-specific. It didn't work because `AshPostgres.MigrationCompileCache` uses an isolated agent abstraction.